recipes/pyflann/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# python source is bundled with flann source
+cd src/python
+cmake . -DLIBRARY_OUTPUT_PATH=$PREFIX/lib -DFLANN_VERSION="$PKG_VERSION"
+$PYTHON setup.py install
```

Comment:
Maybe add python tests? This has been flaky for me sometimes so if you have issues with this we can just forget about it for now.